document_page_group/models/document_page.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class DocumentPage(models.Model):
+
+    _inherit = 'document.page'
+
+    group_ids = fields.Many2many(
+        'res.groups',
+        store=True,
+        relation='document_page_direct_group',
+        column1='document_page_id',
+        column2='group_id',
+        compute='_compute_group_ids'
+    )
+    direct_group_ids = fields.Many2many(
+        'res.groups',
+        relation='document_page_group',
+        column1='document_page_id',
+        column2='group_id',
```

Comment:
I would just a small help for the user, as the names themselves can not be enough

Reply:
Well groups is hidden, but we could define something for direct_group_ids

Reply:
It is not hidden, see in runbot with demo user:
![image](https://user-images.githubusercontent.com/23449160/70065625-b5370080-15eb-11ea-9787-ec5f561e40ef.png)

Reply:
Ups, my mistake then :wink:

Reply:
Now it looks better form the user perspective :+1: